### DISCUSSION TITLE
add emoji to channels

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#53] [Fun Ideas]
Currently, in front of a channel there is either a lock, a globe, the avatar of a dm or a number of participants.

For public channels we could probably have an emoji trying to describe the intent of the channel

another possibility would be to have the emoji related to the category, so you can see at a glance which category that channel belongs to.

**Comment by esethna** [comment]
Ya Slack does this on categories, could be a neat option to offer for channels too! Default could be the globe, but allowing customization makes sense

**Comment by emilyacook** [comment]
+1, I get around this on categories by just copying from emojipedia lol
<img width="216" alt="Screen Shot 2021-06-23 at 3 56 26 PM" src="https://user-images.githubusercontent.com/35075700/123166960-9604fb80-d43b-11eb-8425-5868e3f533cd.png">

**Reply by Willyfrog** [reply]
that's a nice workaround :D

**Reply by esethna** [reply]
OMG this is horrible! We should just add an emoji picker into the category creation/rename modals!!

**Reply by esethna** [reply]
Opened a ticket for this ;) https://mattermost.atlassian.net/browse/MM-36672

**Comment by chetanyakan** [comment]
OR allow uploading channel profile picture icons

**Reply by Willyfrog** [reply]
that's definitely an improvement over the original idea 👍

**Reply by esethna** [reply]
That's interesting. I wonder if that would get DMs and Channels confused in the sidebar since we have profile icons on them now
